<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>311</width>
    <height>212</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Fila</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>20</y>
      <width>56</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>TAMANHO</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>55</y>
      <width>46</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>INSERIR</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>90</y>
      <width>60</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>VERIFICAR</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>127</y>
      <width>41</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>VAZIO?</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Tamanho">
    <property name="geometry">
     <rect>
      <x>81</x>
      <y>20</y>
      <width>133</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Verificar">
    <property name="geometry">
     <rect>
      <x>81</x>
      <y>90</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_eVazio">
    <property name="geometry">
     <rect>
      <x>81</x>
      <y>127</y>
      <width>133</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Inserir">
    <property name="geometry">
     <rect>
      <x>81</x>
      <y>55</y>
      <width>133</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Inserir">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>55</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>INSERIR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Criar">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>20</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>CRIAR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Remover">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>90</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>REMOVER</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Verificar">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>90</y>
      <width>62</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>VERIFICAR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_eVazio">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>127</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>VAZIO?</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>311</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
